[Term Entry] JavaScript Object Function: Object.getPrototypeOf() (#4085)
Codecademy/docs: content/javascript/concepts/objects/terms/getPrototypeOf/getPrototypeOf.md

---

> Title: 'Object.getPrototypeOf()'

Reviewer: ```suggestion
Title: '.getPrototypeOf()'
```

> Description: 'Returns the prototype value of the specified object.'

Reviewer: ```suggestion
Description: 'Returns the prototype value of the specified object'
```

Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Properties'
  - 'Objects'
  - 'Functions'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/create-a-back-end-app-with-javascript'
---

> The **`Object.getPrototypeOf()`** method is a static method that returns the prototype (the value of the internal [[Prototype]] property) of the specified object.

Reviewer: ```suggestion
The **`.getPrototypeOf()`** method is a static method that returns the prototype (the value of the internal [[Prototype]] property) of the specified object.
```
